### DISCUSSION TITLE
Accessible Bootstrap Dropdown

### DISCUSSION THREAD

**Discussion by ledtear214** [twbs/bootstrap#39286] [Q&A]
Is there a way to make the dropdown accessible so that when the parent is navigated to via a keyboard, the dropdown menu is displayed?

**Comment by julien-deramond** [comment]
[Bootstrap dropdowns](https://getbootstrap.com/docs/5.3/components/dropdowns) are already accessible and can be used with keyboards. A specific [Accessibility section](https://getbootstrap.com/docs/5.3/components/dropdowns/#accessibility) adds more detail.
